releases/8.2/languages/fr.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+return [
+    'common_header' => 'PHP 8.2 est une mise à jour majeure du langage PHP. Classes en lecture seule, null, false, and true comme types "stand-alone", dépréciation des propriétés dynamiques, amélioration des performances et plus encore',
+    'documentation' => 'Doc',
+    'main_title' => 'Released!',
+    'main_subtitle' => 'PHP 8.2 est une mise à jour majeure du langage PHP.<br class="display-none-md">Elle contient beaucoup de nouvelles fonctionnalités et d'optimisations, incluant les classes en lecture seule, les types "stand-alone" null, false, et true, la dépréciation des propriétés dynamiques, et plus encore.',
+    'upgrade_now' => 'Migrer à PHP 8.2 maintenant!',
+    'readonly_classes_title' => 'Classes en lecture seule',
+    'dnf_types_title' => 'Types DNF (forme normale disjonctive)',
+    'dnf_types_description' => 'Les types DNF permettent de combiner des types <a href="/manual/en/language.types.type-system.php#language.types.type-system.composite.union">union</a> et <a href="/manual/en/language.types.type-system.php#language.types.type-system.composite.intersection">intersection</a>, en suivant une règle stricte: lorsque des types union et intersection sont combinés, les types intersection doivent être groupés entre crochets.',
```

Comment:
Thanks!

```suggestion
    'dnf_types_description' => 'Les types DNF permettent de combiner des types <a href="/manual/fr/language.types.type-system.php#language.types.type-system.composite.union">union</a> et <a href="/manual/fr/language.types.type-system.php#language.types.type-system.composite.intersection">intersection</a>, en suivant une règle stricte: lorsque des types union et intersection sont combinés, les types intersection doivent être groupés entre crochets.',
```

Aside: maybe it's more useful to use French doc links (here and below).

Reply:
Yep, all doc links now points to the doc fr.